### ISSUE TITLE
is it possible to specify Tablespace on Entity Framework core OnModelCreating

### ISSUE THREAD

**Issue by omora** [oracle/dotnet-db-samples#188]

**Comment by alexkeh** [comment]
No, there isn't. What are you trying to accomplish by specifying a tablespace?

**Comment by omora** [comment]
When we apply the migration to create the database table and indexes it generates:
-- Create table
create table A01ARQ01.ARQ_EMAIL_ADJUNTO
(
  "Id"            RAW(16) not null,
  "NombreFichero" NVARCHAR2(255),
  "Binario"       BLOB,
  "EmailId"       RAW(16) default '00000000000000000000000000000000' not null
)
**tablespace ASSM_ARQAUD_DAT;**
-- Create/Recreate indexes 
create index INX_ARQ_EMAIL_ADJUNTO_01 on ARQ_EMAIL_ADJUNTO (EmailId)
  **tablespace ASSM_ARQAUD_DAT**;
-- Create/Recreate primary, unique and foreign key constraints 
alter table ARQ_EMAIL_ADJUNTO
  add constraint INX_ARQ_EMAIL_ADJUNTO_PK primary key (ID)
  using index 
  **tablespace ASSM_ARQAUD_DAT;**
alter table ARQ_EMAIL_ADJUNTO
  add constraint FK_ARQ_EMAIL_ADJUNTO_ARQ_EMAIL_EmailId foreign key (EMAILID)
  references ARQ_EMAIL (ID) on delete cascade;

It uses the default tablespace for the schema
**Our DBA demand to create index tablespaces in a different one**
create index INX_ARQ_EMAIL_ADJUNTO_01 on ARQ_EMAIL_ADJUNTO (EmailId)
  **tablespace ASSM_ARQAUD_IND**;
and
alter table ARQ_EMAIL_ADJUNTO
  add constraint INX_ARQ_EMAIL_ADJUNTO_PK primary key (ID)
  using index 
  **tablespace ASSM_ARQAUD_IND;**


The work around is not to use migrations and create the tables and index with SQL , the traditional way.
### it's a shame that because of this feature we can't use migrations

**Comment by alexkeh** [comment]
@omora Thanks for brining this to the Oracle EF Core team's attention. I filed an enhancement request (33219678) to have the dev team take a look. 

We will prioritize ERs based on customer interest.

**Comment by tisonv** [comment]
any updates on that matter ?

**Comment by alexkeh** [comment]
@tisonv There has been limited progress. Compared to other feature requests, the community has not upvoted this feature. As of today, there has only been one upvote.

**Comment by alexkeh** [comment]
If your organization needs it, I recommend you vote for it. Other features were upvoted, and they are being prioritized.

Separating data and indexes usually makes little difference to performance. The bigger benefit can come from the DB administration perspective. Some DBAs find it easier to monitor growth and other management tasks by keeping them separate. Of course, this comes down to personal preference.
